src/dashboard/index.ts

```diff
@@ -45,28 +42,24 @@ export default function () {
     const docker = new Docker(mainWindow!)
     const firefox = new Firefox(mainWindow!)
     ipcMain.on('firefox:check', async (_, message) => {
-      
       const firefoxInstalled = await firefox.isInstalled()
       if (!firefoxInstalled) {
-        await firefox.download();
-      }
-      else{
-        await firefox.launch();
+        await firefox.download()
+      } else {
+        await firefox.launch()
       }
     })
 
     ipcMain.on('firefox:lunch', async (_, message) => {
-      await firefox.launch();
+      await firefox.launch()
     })
 
     ipcMain.on('docker:check', async (_, message) => {
       const dockerInstalled = await docker.isInstalled()
       if (!dockerInstalled) {
-        //  await docker.download();
-      }
-      else{
-       // await docker.startCompose()
-       
+        // await docker.download()
+      } else {
```

Comment:
Currently both the `if` and the `else` condition here do not appear to do anything (all code is commented out).

Reply:
I think we can get rid of it completely since we're not using docker anymore. We had this thing just in case.